{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "521fd13d_6b255a3a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1644679
      },
      "writtenOn": "2023-08-25T16:06:08Z",
      "side": 1,
      "message": "The tests still have the 5 failing test cases from b/156167471 before and after this change BTW.",
      "revId": "97848737a9ff67f2c5bd921072e354ff993760ab",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}